lansa-vmss-windows-autoscale-sql-database/azuredeploy.parameters.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+   "parameters": {
+      "msiUri": {
+         "value": "https://lansalpcmsdn.blob.core.windows.net/app/test/AWAMAPP_v14.1.2_en-us.msi"
+      },
+      "vmSku": {
+         "value": "Standard_A2"
+      },
+      "imageOffer": {
+         "value": "lansa-scalable-license"
+      },
+      "vmssName": {
+         "value": "vmsslansa"
```

Comment:
There are some ps scripts in the root of this repo, which I believe do replace these placeholders. The CI definitely replaces them. For debugging I usually keep around a temporary version with my temporary values for the placeholders